resources/styles/global/tutor-booksplash-background.less

```diff
@@ -0,0 +1,44 @@
+// Applies a fixed background effect to a container that includes a large fixed header
+//
+// It's intended to be customized for different looks depending on the type of content
+// the user is styling, i.e. Physics, Biology, etc
+.tutor-booksplash-background {
+
+  &:before {
+    content: attr(data-title);
+    position: fixed;
+    left: 0px;
+    right: 0px;
+    overflow: hidden;
+    background-color: @tutor-tertiary;
+    color: @tutor-secondary;
+    text-align: center;
+    font-size: 300px;
+    line-height: 200px;
+    font-family: 'HelveticaNeue-Bold', 'Helvetica Neue Bold', 'Helvetica Neue';
```

Comment:
I'm not sure.  This is the only place that I see Helvetica being used. `grep -r Helvetica` only returns this at least.  Is still worthwhile to have it available globally do you think?